src/main/java/com/philips/research/bombase/core/npm/domain/NpmAPI.java

```diff
@@ -38,8 +36,8 @@
         DistJson dist;
 
         @Override
-        public int score(Field field) {
-            return NPM_SCORE;
+        public Trust trust(Field field) {
+            return Trust.LIKELY;
```

Comment:
We can experiment with changing the level of Trust depending on the presence of certain fields. LIKELY seems fair now.

Reply:
"Trust" should depend on the likelyhood that a field contains the correct value; not if the field exists. (Non-existing fields are not written, and will therefore never override existing values.)

Current assumption is that package managers know better than ClearlyDefined, but declared licenses in ClearlyDefined could be curated and therefore get priority. (But this is mostly based on gut feeling...)